packages/phoenix-ng/projects/phoenix-ui-components/lib/components/ui-menu/io-options/io-options-dialog/io-options-dialog.component.test.ts

```diff
@@ -148,14 +154,16 @@ describe('IoOptionsDialogComponent', () => {
   it('should handle zipped event data', async () => {
     const zip = new JSZip();
     zip.file('test_data.json', '{ "event": null }');
-    const jivexmlData = await fetch('assets/test_data/JiveXML.xml');
+    const jivexmlData = await fetch(
+      'https://github.com/HSF/phoenix/blob/master/packages/phoenix-ng/projects/phoenix-app/src/assets/files/JiveXML/JiveXML_336567_2327102923.xml'
```

Comment:
```
const jivexmlData = await fetch('assets/test_data/JiveXML.xml');
```

We are using GitHub URL here? 🤔 There is some problem fetching using absolute paths in Jest and since `fetch` is not defined in Node.js, it will require a mock or a polyfill (`whatwg-fetch`, `node-fetch`, `jest-fetch-mock`, to name a few). Still, what can be better?

Reply:
Got it.